demos/polls/aiohttpdemo_polls/routes.py

```diff
@@ -12,6 +12,10 @@ def setup_routes(app):
     app.router.add_get('/poll/{question_id}/results',
                        results, name='results')
     app.router.add_post('/poll/{question_id}/vote', vote, name='vote')
+    setup_static_routes(app)
+
+
+def setup_static_routes(app):
     app.router.add_static('/static/',
                           path=str(PROJECT_ROOT / 'static'),
```

Comment:
IIRC `path` parameter accepts `pathlib.Path` now.
Would you remove `str()` call and check?

Reply:
I've just tested it. You're right, str is not needed. Removing it